scripts/insolard/genesis.yaml

```diff
@@ -106,11 +106,11 @@ discovery_nodes:
 #    cert_name: "discovery_cert_23.json"
 
 nodes:
-  -
-    host: "127.0.0.1:63836"
-    role: "virtual"
-    keys_file: "scripts/insolard/nodes/1/keys.json"
-    cert_name: "node_cert_1.json"
+#  -
```

Comment:
It should be uncommented. Because this part of the file provides info about a node, that connects to network after a consensus phase.

Reply:
done